cmd/project.go

```diff
@@ -586,8 +587,17 @@ Examples:
 
 		})
 
-		// run the git flow
-		projectGitFlow(ctx, provider, tmplContext, githubAction)
+		gitInfo, err := deployer.GetGitInfoRecursive(logger, projectDir)
+
+		logger.Debug("git info: %+v", gitInfo)
+
+		if err != nil {
+			logger.Info("failed to get git info: %s", err)
+		}
```

Comment:
```suggestion
		gitInfo, err := deployer.GetGitInfoRecursive(logger, projectDir)
		if err != nil {
			logger.Info("failed to get git info: %s", err)
		}
		logger.Debug("git info: %+v", gitInfo)
```

https://tenor.com/bjl8N.gif